specifyweb/frontend/js_src/lib/components/DataModel/saveBlockers.tsx

```diff
@@ -113,42 +111,26 @@ export function useSaveBlockers(
 export function setSaveBlockers(
```

Comment:
are blockers ever set for more than one field in a single function call?
if not, then why not just prepend the blocker key with the field name automatically inside of setSaveBlockers rather than forcing every place to call getFieldBlockerKey (or risk forgetting to call it and thus creating a bug)